README.md

```diff
@@ -1,31 +1,33 @@
 [![Documentation](https://img.shields.io/badge/doc-reference-blue)](https://docs.restate.dev)
 [![Examples](https://img.shields.io/badge/view-examples-blue)](https://github.com/restatedev/examples)
 [![Discord](https://img.shields.io/discord/1128210118216007792?logo=discord)](https://discord.gg/skW3AZ6uGd)
-[![Twitter](https://img.shields.io/twitter/follow/restatedev.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=restatedev)
+[![Slack](https://img.shields.io/badge/Slack-4A154B?logo=slack&logoColor=fff)](https://join.slack.com/t/restatecommunity/shared_invite/zt-2v9gl005c-WBpr167o5XJZI1l7HWKImA)
+[![Twitter](https://img.shields.io/twitter/follow/restatedev.svg?style=social&label=Follow)](https://x.com/intent/follow?screen_name=restatedev)
 
 # Restate - Building resilient applications made easy!
 
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://restate.dev/poster_intro_dark.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://restate.dev/poster_intro_2.svg">
-    <img alt="Restate overview" src="https://restate.dev/poster_intro_2.svg" width="500">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-light.png">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-dark.png">
+    <img alt="Restate overview" src="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-dark.png" width="500">
```

Comment:
Nit: it looks like there's a loose arrow below tracing - maybe that should be one-way, too?

![image](https://github.com/user-attachments/assets/f1cead5d-7236-41f8-992c-4f190b3b3cbb)